packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -470,11 +469,22 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
         hidden={persistMenu ? menuHidden : undefined}
         className={css('ms-BaseButton-menuhost', menuProps.className)}
         target={this._isSplitButton ? this._splitButtonContainer.current : this._buttonElement.current}
-        onDismiss={onDismiss}
+        onDismiss={this._onDismissMenu}
       />
     );
   };
 
+  private _onDismissMenu = (ev: any): void => {
```

Comment:
yes, this ev is an any in the typings.

Reply:
Sounds like we should fix upstream since we own those typings, don't we?

Reply:
I agree on a general level, we could definitely have the private `_onDismissMenu` take the correct typings, but we wouldn't be able to change the public `onDismissMenu` since it could potentially break customers.

Reply:
@cliffkoh I think we should let this PR go in if Micah doesn't respond before the but merges it. In that case I'll change the typings in a separate PR because it would be a non-public facing api change.

Reply:
I'm ok with that :) henc ei didn't remove automerge label or block. Thanks for volunteering to do the follow up!

Reply:
@khmakoto Oh i just saw your first response. We don't consider types tightening to be a breaking change. Improving our types and catching actual bugs is considered a good thing! We have communicated and solicited feedback on this policy last year and none of our partner teams expressed opposition or concerns about this policy.

Reply:
Sounds good, I'll send a followup PR tomorrow then. Thanks for the context @cliffkoh!

Reply:
Agree with the above. Types tightening has not been met with friction in my experience.

Reply:
thanks for letting this go through and doing the typings separately. This PR is unblocking a customer.